pom.xml

```diff
@@ -211,7 +211,7 @@ flexible messaging model and an intuitive client API.</description>
     <jakarta.activation.version>1.2.2</jakarta.activation.version>
     <jakarta.xml.bind.version>2.3.3</jakarta.xml.bind.version>
     <jakarta.validation.version>2.0.2</jakarta.validation.version>
-    <jna.version>5.12.1</jna.version>
+    <jna.version>4.2.0</jna.version>
```

Comment:
I guess you'd prefer revert this change as described in https://github.com/apache/pulsar/pull/17262?

Reply:
oh thanks, that's what I call a bad git rebase